Automattic/jetpack
# Holiday Snow: switch to a CSS-only implementation

Fixes #40569

## Proposed changes:

> [!NOTE]
> We can tweak the appearance of the snow a bit more if necessary.

Moving away from Javascript:

- should be better for performance
- will avoid glitches like the one described in #40569
- will avoid issues with other parts of the site, such as notifications or comments. (see p1734109348431089-slack-C029GN3KD )
- allows us to use Holiday Snow on p2s.
- CPU usage remains low, this is not a change over the existing solution. (tested in Firefox).


### Other information:


## Jetpack product discussion

* p1734109348431089-slack-C029GN3KD

## Does this pull request change what data or activity we track or use?

* No

## Testing instructions:

* On a WoA or Simple site, go to Settings > General and enable snow
* Go to the frontend and check the look of the snow.
* Attempt to open the notifications bar or leave a comment ; snow should continue to fall.
